tests/regression/lp-1644439/kernel-new-enough.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Why is this in a file by itself?  It's literally one line (the "if" is unnecessary since the uname will already exit 1).